test/indexSpec.js

```diff
@@ -435,9 +434,16 @@ describe('PluginBells', function () {
       })
     })
 
+    describe('getPrefix', function () {
+      it('returns the plugin\'s prefix', function * () {
+        const prefix = yield this.plugin.getPrefix()
```

Comment:
`yield` will also accept generators. This test (and similar ones) should probably check to ensure that a promise is being returned. (Would be easy enough for us to forget to put a co.wrap somewhere.)

Would recommend using `chai-as-promised` and writing the test as:
``` js
it('returns the plugin\'s prefix', function * () {
  const prefixPromise = this.plugin.getPrefix()
  yield assert.eventually.equal(prefixPromise, 'example.red')
})
```

cc/ @bensharaf